tests/e2e/cucumber/features/keycloak/groups.feature

```diff
@@ -0,0 +1,79 @@
+
+
+  Scenario: keycloak group sync with oCIS
+    Given "Admin" creates following user using API
+      | id    |
+      | Alice |
+      | Brian |
+    And "Alice" creates the following files into personal space using API
+      | pathToFile          | content              |
+      | shareToSales.txt    | Keycloak group share |
+      | shareToSecurity.txt | Keycloak group share |
+
+    When "Admin" logs in
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the groups management page
+    When "Admin" creates the following groups
+      | id       |
+      | security |
+      | finance  |
+      | sales    |
+    Then "Admin" should see the following group
+      | group            |
+      | security         |
+      | finance          |
+      | keycloak sales   |
+      | keycloak finance |
+
+    When "Admin" navigates to the users management page
+    And "Admin" adds the user "Brian" to the groups "finance,keycloak sales" using the sidebar panel
+    And "Admin" logs out
+
+    And "Alice" logs in
+    And "Alice" shares the following resource using the sidebar panel
+      | resource            | recipient      | type  | role     | resourceType |
+      | shareToSales.txt    | keycloak sales | group | Can edit | file         |
+      | shareToSecurity.txt | finance        | group | Can edit | file         |
+    And "Alice" logs out
+
+    And "Brian" logs in
+    And "Brian" navigates to the shared with me page
+    # user should have access to unsynced shares
+    When "Brian" opens the following file in texteditor
+      | resource         |
+      | shareToSales.txt |
+    And "Brian" closes the file viewer
+    And "Brian" edits the following resources
+      | resource            | content     |
+      | shareToSecurity.txt | new content |
+    And "Brian" logs out
+
+    When "Admin" logs in
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the groups management page
+    # Renaming a Keycloak group results in the creation of a new group on the oCIS server (see https://github.com/owncloud/ocis/issues/10445).
+    # After renaming a group, it may take up to 5 minutes for the changes to sync, so avoid using the renamed group in the subsequent steps.
+    And "Admin" changes displayName to "a renamed group" for group "keycloak finance" using the sidebar panel
+
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the groups management page
+    When "Admin" deletes the following group using the context menu
+      | group |
+      | sales |
+    Then "Admin" should not see the following group
+      | group |
+      | sales |
+    When "Admin" deletes the following groups using the batch actions
+      | group    |
+      | security |
+      | finance  |
+    Then "Admin" should not see the following groups
+      | group    |
+      | security |
+      | finance  |
```

Comment:
why do we need these ocis group steps? one should be enough if you insist

Reply:
to test keycloak integration doesn't affect oCIS side group